src/utils/get-page-schema.js

```diff
@@ -0,0 +1,51 @@
+export const getPageSchema = (url, articleInfo = null) => {
+    const schema = {
+        '@context': 'http://schema.org',
+        url: url,
+    };
+    const siteImageObject = {
+        '@type': 'imageObject',
+        url:
+            'https://webassets.mongodb.com/_com_assets/cms/mongodb_logo1-76twgcu2dm.png',
```

Comment:
I'm assuming this is also the mongo logo to reference here (same as in the blog).

Reply:
Yea, you can grab the logo from the ticket and import it in this file